### ISSUE TITLE
Also distribute your app via fdroid.org

### ISSUE THREAD

**Issue by c33s** [Findus23/kurzparkzonen-wien#1]
https://f-droid.org/en/contribute/

**Comment by Findus23** [comment]
Hi,

I'm a huge fan of fdroid and use it exclusively on my phone, but I think it's not possible to publish the app in the main fdroid repository. It depends on the built website and even though both the app and the web-app are fully FOSS, I don't think the fdroid buildservers can build any Javascript.

If you want to, I can publish a fdroid repo with the built APKs on the website.

**Comment by c33s** [comment]
i am not sure i understand, only the apk should be build by fdroid. so maybe you need to split up the repo to have one for the webpage and one for the android app.

publish an own froid repo is nice but to benefit from the good reputation of fdroid your app should be listed there.

**Comment by Findus23** [comment]
I'm not completely familiar with the rules for fdroid apps, but I know that everything needs to build from source.

As the app mostly consist of only [one file](https://github.com/Findus23/kurzparkzonen-wien/blob/master/android/app/src/main/java/at/lw1/kurzparkzonen/wien/MainActivity.java) that shows a webview of the webapp (that is bundled in the app so it works offline). So to build this from source, also the webapp needs to be build too. While this isn't hard, this needs a lot of javascript build tools.
And I can't find a lot of information if this is supported now:
https://gitlab.com/fdroid/fdroidserver/issues/60

If you want to you can try it out and create a issue in https://gitlab.com/fdroid/rfp/issues and I'll help make it compatible.

**Comment by Findus23** [comment]
In the meantime there is now a repo at 
https://kurzparkzonen-wien.lw1.at/fdroid/repo/?fingerprint=7C17025400B54D81E1673394A6F3989D5460B1555417951835064B6EE31584C9

**Comment by Findus23** [comment]
Interestingly there was a very similarly built app published on fdroid recently:

https://gitlab.com/fdroid/rfp/issues/1121
https://f-droid.org/en/packages/app.fedilab.openmaps/

**Comment by sjehuda** [comment]
Helo! Is there any progrress on this?

**Comment by Findus23** [comment]
Not really. Nobody mentioned it to me in the last years and I think it isn't that important as the app doesn't really have many benefits over just visiting https://kurzparkzonen-wien.lw1.at/ in the browser.
